.changeset/alle-headings.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
deze file is niet nodig voor de 1.0.0 publish (er zou meteen een bump naar 2.0.0 voor alle componenten gedaan worden op deze manier)